packages/gatsby-plugin-preact/src/gatsby-node.js

```diff
@@ -24,6 +24,25 @@ exports.onCreateWebpackConfig = ({ stage, actions, getConfig }) => {
     })
   }
 
+  // add preact to the framework bundle
+  if (stage === `build-javascript`) {
+    const webpackConfig = getConfig()
+    if (
+      webpackConfig?.optimization?.splitChunks?.cacheGroups?.framework?.test
```

Comment:
Can we use optional chaining in gatsby-node?

Reply:
We transpile `src` and `babel-preset-gatsby-package` handles optional chaining, so we are good here